### PR TITLE
doc: correct PVC expansion documentation

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: docs/expand-pvc.md [hunk 1]
```diff
@@ -10,13 +10,13 @@
 
 ## Prerequisite
 
-- For filesystem resize to be supported for your kubernetes cluster, the
+- For filesystem expansion to be supported for your kubernetes cluster, the
   kubernetes version running in your cluster should be >= v1.15 and for block
   volume resize support the kubernetes version should be >=1.16. Also,
   `ExpandCSIVolumes` feature gate has to be enabled for the volume resize
```
Comment (Madhu-1) [inline docs/expand-pvc.md:16]: changes resize to expand in other places also

Comment (humblec) [reply docs/expand-pvc.md]: @Madhu-1 sharp.. thanks.. updated in the other places too.

**Comment by humblec** [comment]
@Madhu-1 can we add ready-to-merge label here?

File: docs/expand-pvc.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ items:
     conditions:
     - lastProbeTime: null
       lastTransitionTime: "2019-12-19T05:49:39Z"
-      message: Waiting for user to (re-)start a pod to finish file system resize of
+      message: Waiting for user to (re-)start a pod to finish file system expand of
```
Comment (Madhu-1) [inline docs/expand-pvc.md:118]: it should not be changed here see https://github.com/kubernetes-csi/external-resizer/blob/9805ed39a9fe7c533034619b79faef26d1b3afb0/pkg/controller/controller.go#L475

File: docs/expand-pvc.md [hunk 3]
```diff
@@ -238,16 +238,16 @@ status:
   conditions:
   - lastProbeTime: null
     lastTransitionTime: "2019-12-19T06:02:15Z"
-    message: Waiting for user to (re-)start a pod to finish file system resize of
+    message: Waiting for user to (re-)start a pod to finish file system expand of
```
Comment (Madhu-1) [inline docs/expand-pvc.md:241]: same here as well.

**Comment by humblec** [comment]
@Madhu-1 addressed the comments. PTAL . thanks !

File: docs/development-guide.md [hunk 4]
```diff
@@ -112,9 +112,9 @@ questions: what changed and why? The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```text
-cephfs: update cephfs resize
+cephfs: update cephfs expand
 
-use cephfs resize to resize subvolume
+use cephfs expand to expand subvolume
```
Comment (Madhu-1) [inline docs/development-guide.md:117]: there is no cephfs expand command

Comment (humblec) [reply docs/development-guide.md]: Done. Thanks @Madhu-1

File: docs/development-guide.md [hunk 5]
```diff
@@ -112,9 +112,9 @@ questions: what changed and why? The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```text
-cephfs: update cephfs resize
+cephfs: update cephfs expand
```
Comment (Madhu-1) [inline docs/development-guide.md:115]: I think we can leave it, no need to update this is specific to cephfs command not the CSI specification. can you revert it back?

Comment (humblec) [reply docs/development-guide.md]: @Madhu-1 done. ptal

File: docs/development-guide.md [hunk 6]
```diff
@@ -114,7 +114,7 @@ the body of the commit should describe the why.
 ```text
 cephfs: update cephfs resize
 
-use cephfs resize to resize subvolume
+use cephfs resize to expand subvolume
```
Comment (Madhu-1) [inline docs/development-guide.md:118]: @humblec this is not reverted back yet

Comment (humblec) [reply docs/development-guide.md]: @Madhu-1 done. now.. :+1:

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

**Comment by humblec** [comment]
@nixpanic @Madhu-1 may be add `ready-to-merge` label?

**Pull request merged**